cpplib/src/main/scala/scala/scalanative/cpp/CppObject.scala

```diff
@@ -0,0 +1,18 @@
+package scala.scalanative.cpp
```

Comment:
At the moment we have no plans in supporting C++ interoperability as a feature in SN. It's fine to use C++ to implement SN internals, but it should always remain in `scalanative.runtime` or be otherwise private and don't provide any APIs that users might rely on.

Reply:
No problem!